ligo/stablecoin/spec.ligo

```diff
@@ -173,18 +179,41 @@ type permissions_descriptor is michelson_pair_right_comb(permissions_descriptor_
 type permissions_descriptor_params is
   contract (permissions_descriptor)
 
-type mint_params is record
-  to_    : address
-; amount : nat
+type pause_params is unit
+
+type unpause_params is unit
+
+type configure_minter_params_ is record
+  minter                    : address
+; current_minting_allowance : option (nat)
+; new_minting_allowance     : nat
 end
 
-type burn_params is record
-  from_  : address
+type configure_minter_params is michelson_pair_right_comb (configure_minter_params_)
+
+type remove_minter_params is address
+
+type mint_param_ is record
+  to_    : address
 ; amount : nat
 end
 
+type mint_param is michelson_pair_right_comb (mint_param_)
+
+type mint_params is list (mint_param)
+
+type burn_params is list (nat)
+
+type transfer_ownership_param is address
+
+type accept_ownership_param is unit
+
+type change_master_minter_param is address
+
+type change_pauser_param is address
+
 type parameter is
-  Transfer               of transfer_params
+  Transfer               of transfer_param
```

Comment:
@awkure As per FA2 parameter of transfer is still `list(transfer_param)` right? So shouldn't it be `transfer_params` here ?

Reply:
No, the list is now enclosed in `parameter.destinations`

Reply:
This is what we have now:
```
                               (pair %transfer
                                  (address %from_)
                                  (list %destinations (pair (address %to_) (pair (nat %token_id) (nat %amount))))))
```

Compare it with FA2, it's different: https://gitlab.com/tzip/tzip/-/blob/master/proposals/tzip-12/tzip-12.md#transfer

Reply:
Also `%destinations` should be changed to `%txs`.